os/kernel/log_dump/log_dump.c

```diff
@@ -62,6 +63,10 @@
 
 #define LOG_CHUNK_SIZE			sizeof(struct log_dump_chunk_s)
 
+#define LOG_DUMP_HANG_CHECK_SEC		60	/* 60s */
```

Comment:
We use 3 mins for watchdog. How about 2 and half mins?
I suggest this because this is for debugging.

And no need to cover them with debug new feature?

Reply:
I have confirmed the case where an assert occurs because it exceeds the post max of another IRQ even though it has been less than two minutes since the hang occurred.

In this case, it is also possible to debug with assert, but debugging may be difficult if a hang occurs because the compression thread in log_dump is not completed.

How about reducing the time to one minute and thirty seconds for this?

If compression is performed normally, the priority of the compression thread is high, so the actual counting period will delay. This count only works normally when the compression thread stops.

Reply:
Ok, Anyway let's leave some explanation of those values.